indico/modules/events/schemas.py

```diff
@@ -0,0 +1,20 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2019 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from indico.core.marshmallow import mm
+from indico.modules.events.models.events import Event
+
+
+class EventSchema(mm.ModelSchema):
```

Comment:
I don't like this name for a schema that simple. I'd rather have a `PaperEventSchema` inside the papers module that defines this (and if we end up needing this particular schema in multiple places we can always move it back with a more generic name like `LockedEventSchema`)